utils/build/docker/ruby/graphql23/config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,26 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Small nit.
If this file is not necessary for the Rails app to work, can we remove it ?